documentation/src/pages/sveltekit/start-here/quick-start.md

```diff
@@ -51,50 +51,62 @@ export const auth = lucia({
 
 ## 3. Sign up page
 
-Create `routes/signup/` route dir. This route will handle account creation. Create 3 folders inside it: `+page.svelte`, `+page.server.ts`, and `+page.ts`.
+Create `routes/signup/` route dir. This route will handle account creation. Create 2 files inside it: `+page.svelte` and `+page.server.ts`.
 
 ### Sign up form
 
 This form will have an input field for username and password.
 
 ```svelte
+<!-- routes/signup/+page.svelte -->
 <script lang="ts">
-	import { enhance } from "$app/forms";
+	import { enhance } from '$app/forms';
 </script>
 
 <div>
 	<h1>Create an account</h1>
-	<form method="post" use:enhance>
-		<label for="username">username</label><br />
+	<form method="POST" use:enhance>
+		<label for="username">Username</label><br />
 		<input id="username" name="username" /><br />
-		<label for="password">password</label><br />
+		<label for="password">Password</label><br />
 		<input type="password" id="password" name="password" /><br />
-		<input type="submit" value="Continue" class="button" />
+		<input type="submit" value="Signup" />
 	</form>
 </div>
 ```
 
 ### Create users
 
-We'll set the provider id as `username` and the username as the identifier. This tells Lucia that user was created using username/password auth method and the unique identifier is the username. Let's also set the password and store the username. After creating a new user, create a new session and store the session id as a cookie.
+We'll set the provider id as `username` and the username as the identifier. This tells Lucia that the user was created using the username/password auth method and that the unique identifier is the username. We'll also set the password and store the username. After creating a new user, create a new session and store the session id as a cookie.
 
-For the session to update in the client, we need to call [`invalidateAll()`](https://kit.svelte.dev/docs/modules#$app-navigation-invalidateall) or refresh the page entirely. `use:enhance` will only call `invalidateAll()` when the server returns a success response, and a redirect response is not considered as a success response. Since we're just using the default behavior of `use:enhance`, the action will not return a redirect, and the load function (explained below) will handle redirect after sign up.
+For the session to update in the client, we need to call [`invalidateAll()`](https://kit.svelte.dev/docs/modules#$app-navigation-invalidateall) or refresh the page entirely. `use:enhance` will only call `invalidateAll()` when the server returns a success response (a redirect response is not considered as a success response). Since we're just using the default behavior of `use:enhance`, the action will not return a redirect, and the load function will handle redirect after sign up.
```

Comment:
I think this part should mention the load function and how it’ll be triggered on invalidateAll and on navigation